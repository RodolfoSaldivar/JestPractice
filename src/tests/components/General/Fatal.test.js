import React from 'react';
import { shallow } from 'enzyme';
import Fatal from '../../../components/General/Fatal';

describe('<Fatal />', () => {
	const fatal = shallow(<Fatal />);

	it('Renders correctly', () => {
		expect(fatal).toMatchSnapshot();
	});

	it('Has <h2 /> with correct classes', () => {
		const h2 = fatal.find('h2');
		expect(h2).toHaveLength(1);
		expect(h2.prop('className')).toEqual('center rojo');
	});
});
