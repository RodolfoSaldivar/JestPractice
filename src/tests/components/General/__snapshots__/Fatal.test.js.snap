// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Fatal /> Renders correctly 1`] = `
<h2
  className="center rojo"
/>
`;
